presto-orc/src/main/java/com/facebook/presto/orc/writer/MapFlatColumnWriter.java

```diff
@@ -125,6 +125,8 @@ public class MapFlatColumnWriter
     // This field contains cached stripe column statistics for the value node(s)
     // for this edge case.
     private Map<Integer, ColumnStatistics> emptyValueColumnStatistics;
+    private long rawSize;
```

Comment:
This writer does not use column statistics builder, it creates statistics directly, that's why I'm independently collecting rawSize and stripeRawSize.